### PR TITLE
fix camera permission handle in android below 6.0

### PR DESCRIPTION
**Pull request by punksta**
fix https://github.com/react-native-community/react-native-camera/issues/1094

### CODE REVIEW

File: src/Camera.js [hunk 1]
```diff
@@ -191,10 +191,18 @@ export default class Camera extends Component {
         message: this.props.permissionDialogMessage,
       });
 
+      // On devices before SDK version 23, the permissions are automatically granted if they appear in the manifest, 
+      // so check and request should always be true.
+      // https://github.com/facebook/react-native-website/blob/master/docs/permissionsandroid.md
+      const isAuthorized = Platform.Version >= 23
+		      ? granted === PermissionsAndroid.RESULTS.GRANTED
+          : granted === true;
```
Comment (sibelius) [inline src/Camera.js]: can we do something simpler, like this:

`granted === PermissionsAndroid.RESULTS.GRANTED || granted === true` ?

Comment (punksta) [reply src/Camera.js]: with `Platform.Version >= 23` it looks more strict for me.

**Comment by sibelius** [comment]
it makes sense, tks

**Comment by punksta** [comment]
Thanks for fast reply!

**Comment by HexiChen** [comment]
It helps me,ths a lot

**Pull request merged**